<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLic</class>
 <widget class="QDialog" name="DialogLic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>500</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>730</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>730</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Azioni utenti</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>710</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>710</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>710</width>
     <height>320</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>710</width>
     <height>320</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>710</width>
     <height>420</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>159</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>318</number>
   </attribute>
   <column>
    <property name="text">
     <string>Client</string>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prov</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Qt√†</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Attivo</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Scade</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Req</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lic</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Server-ID</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Active</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelClient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cliente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClient">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>10</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>37</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>37</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="labelEmail">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>63</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmail">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>63</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxProv">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>90</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="labelProv">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>90</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Provincia</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelQty">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Quantit√†</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQty">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditActive">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>37</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelActive">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>39</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Dal</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelExpire">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>67</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Fino</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditExpire">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>65</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btGenerate">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>70</y>
     <width>30</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btSave">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>30</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClient_2">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelQty_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Req</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClient_3">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>37</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelQty_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>37</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Lic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btDelete">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGenerate">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>73</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Genera</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSave">
   <property name="geometry">
    <rect>
     <x>627</x>
     <y>73</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Salva</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDelete">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>453</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditClient</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>comboBoxProv</tabstop>
  <tabstop>lineEditQty</tabstop>
  <tabstop>dateEditActive</tabstop>
  <tabstop>dateEditExpire</tabstop>
  <tabstop>lineEditClient_2</tabstop>
  <tabstop>lineEditClient_3</tabstop>
  <tabstop>btGenerate</tabstop>
  <tabstop>btSave</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>btDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
